src/components/CircularProgress/CircularProgress.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useId } from 'react';
```

Comment:
I know this project uses React 18, but we cannot use v18 features yet unfortunately, since we are stuck on v17 for the time being in other projects where we use these components. 

Ill try and make this clear in the documentation so we avoid this confusion later